plenum/common/txn_util.py

```diff
@@ -78,7 +78,7 @@ def reqToTxn(req: Request, cons_time=None):
         f.IDENTIFIER.nm: data[f.IDENTIFIER.nm],
         f.REQ_ID.nm: data[f.REQ_ID.nm],
         f.SIG.nm: data[f.SIG.nm],
-        f.SIGS.nm: data[f.SIGS.nm],
```

Comment:
Should we use `get` for SIG as well (since we do not set it in Request.as_dict)?

Reply:
probably yes